source/configuration/program-detail-page.md

```diff
@@ -9,9 +9,9 @@ The Program Detail Page displays detailed, program-level information for each of
 
 1. The files that specify the configuration parameters of the Bento Landing Page are stored in the GitHub `https://github.com/CBIIT/bento-frontend` (representing your GitHub username as `YOUR-USERNAME`). Create a local clone of your fork into a local directory, represented in these instructions as `$(src)`.
 
-2. Configuration Parameters for all Program Detail Page elements can be specified in the file: `$(src)/bento-frontend/blob/master/src/bento/programDetailData.js`.
+2. Configuration Parameters for all Program Detail Page elements can be specified in the file: `$(src)/packages/bento-frontend/src/bento/programDetailData.js`.
 
-3. All images and icons that you use in your Bento instance should be accessible via a public url. 
+3. All images and icons that you use in your Bento instance should be accessible via a public url.
```

Comment:
Let's try not to use "you", so instead we can say, All images and icons used a Bento instance....